Cargo.toml

```
@@ -604,7 +613,9 @@ lto = false
 # `geth` responds to invalid payloads with objects containing `method` and `params`.
 # We had to fork `jsonrpc` because it does not allow nonstandard members.
 jsonrpc-core = { git = 'https://github.com/grandinetech/jsonrpc.git' }
-sha2 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'sha2-v0.10.9-up.0' }
+# sha2 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'sha2-v0.10.9-up.0' }
+sha2 = { git = 'https://github.com/jimmychu0807/grandine-universal-precompiles.git', branch = 'RustCrypto-hashes/sha2-v0.10.9' }
```

Once https://github.com/grandinetech/universal-precompiles/pull/3 is merged, modify to

```suggestion
sha2 = { git = "https://github.com/grandine/universal-precompiles.git", branch = "RustCrypto-hashes/sha2-v0.10.9" }
```